lib/barnes/resource_usage.rb

```diff
@@ -28,8 +28,8 @@ class ResourceUsage < Panel
     def initialize(sample_rate)
       super()
 
-      require 'barnes/instruments/puma_backlog'
-      backlog_reporter = Barnes::Instruments::PumaBacklog.new
+      require 'barnes/instruments/puma_instrument'
+      backlog_reporter = Barnes::Instruments::PumaInstrument.new
```

Comment:
`backlog_reporter` as a name probably doesn't make sense anymore.

Reply:
ha, good catch. Will update in on second.